### PR TITLE
fix: mark template as public

### PR DESCRIPTION
**Pull request by rupert-mckay**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (rupert-mckay) [inline .eslintrc.json:1]: Spaces have changed to tabs in this file.

Also pointed the eslint at the renamed config package

File: package.json [hunk 2]
```diff
@@ -1,47 +1,50 @@
...
+	],
+	"main": "./dist/your-project-name.umd.cjs",
+	"module": "./dist/your-project-name.js",
+	"exports": {
+		".": {
+			"import": "./dist/your-project-name.js",
+			"require": "./dist/your-project-name.umd.cjs"
+		}
+	},
+	"publishConfig": {
```
Comment (rupert-mckay) [inline package.json:26]: Changed spaces to tabs.

The only other change is this `publishConfig` section which tells semantic-release to tell npm to make this project public when published.

**Review by artursvonda — APPROVED** [review]
Just to double check, the changes to lock file are formatting, correct? And those changes will be automatic? Just making sure it's not specific IDE making changes instead of those fixes being systematic.

**Comment by rupert-mckay** [comment]
Prettier hadn't been run on some files in the template. Now it has.

The lock file has been updated due to pointing at the newer renamed eslint config.

**Comment by rupert-mckay** [comment]
Nope. Happy to revert that part of the change if we want `npm run lint:fix` to be the single source of formatting truth.

**Comment by artursvonda** [comment]
I think the `eslint` (or, more generally, `npm run lint` and `npm run lint:fix`) should be source of truth. It's also ok to add those json files to eslint config to ensure it's handled. But I don't like idea of this change being because of specific user setup.

**Comment by rupert-mckay** [comment]
Mark it as requesting changes then. (I would do it myself but GitHub forbids me from doing so on a PR in which I am an author).

**Review by artursvonda — APPROVED** [review]

**Pull request merged**